### PR TITLE
Merge the latest code into master

### PR DESCRIPTION
**Pull request by jefftlin**
### What is the purpose of the change
(For example: Exchangis-Job defines the core ability of Exchangis, it provides the abilities of job management, job transform, and job launch.
Related issues: #50. )

### Brief change log
- Feature upgrades from 1.0.0 to 1.1.2

### Verifying this change
- Added tests for creating and execute the Exchangis jobs and verify the availability of different Exchangis Job, such as sqoop job, datax job.

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (yes)
- Anything that affects deployment: (yes)
- The Core framework, i.e., JobManager, Server.: (no)

### Documentation
- Does this pull request introduce a new feature? (yes)
- If yes, how is the feature documented? (docs)

### CODE REVIEW

File: docs/en_US/ch1/exchangis_appconn_deploy_en.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ After the exchangis-appconn is installed and deployed, the following steps can b
 2.	 Check whether the project is created synchronously on Exchangis. Successful creation means successful installation of appconn 
 ![image](https://user-images.githubusercontent.com/27387830/169782337-678f2df0-080a-495a-b59f-a98c5a427cf8.png)
 
-For more operation, please refer to [Exchangis 1.0 User Manual](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0/docs/en_US/ch1/exchangis_user_manual_en.md)
+For more operation, please refer to [Exchangis 1.0 User Manual](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0-rc/docs/zh_CN/ch1/exchangis_user_manual_cn.md)
```
Comment (wushengyeyouya) [inline docs/en_US/ch1/exchangis_appconn_deploy_en.md:70]: Please modify it.

File: docs/zh_CN/ch1/exchangis_deploy_cn.md [hunk 2]
```diff
@@ -271,12 +293,16 @@ Exchangis 已默认提供了编译好的前端安装包，可直接下载使用
...
+如您想正常使用 Exchangis1.1.2 前端，还需安装 DSS ExchangisAppConn 插件，请参考: [ExchangisAppConn 插件安装文档](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.1.2/docs/zh_CN/ch1/exchangis_appconn_deploy_cn.md)
 
 ## 4. Linkis Sqoop 引擎安装部署
 
-如您想正常执行 Exchangis1.0.0 的 Sqoop作业，还需安装 Linkis Sqoop 引擎，请参考: [Linkis Sqoop 引擎插件安装文档](https://linkis.apache.org/zh-CN/docs/latest/engine-usage/sqoop)
+如您想正常执行 Exchangis1.1.2 的 Sqoop作业，还需安装 Linkis Sqoop 引擎，请参考: [Linkis Sqoop 引擎插件安装文档](https://linkis.staged.apache.org/zh-CN/docs/1.4.0/engine_usage/sqoop)
+
+## 5. Linkis DataX 引擎安装部署
+
+如您想正常执行 Exchangis1.1.2 的 DataX作业，还需安装 Linkis DataX 引擎，请参考: [Linkis DataX 引擎插件安装文档](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.1.2/docs/zh_CN/ch1/exchangis_datax_deploy_cn.md)
```
Comment (wushengyeyouya) [inline docs/zh_CN/ch1/exchangis_deploy_cn.md:304]: please modify it.

File: docs/zh_CN/ch1/exchangis_deploy_cn.md [hunk 3]
```diff
@@ -271,12 +293,16 @@ Exchangis 已默认提供了编译好的前端安装包，可直接下载使用
...
 
-## 5. 如何登录使用 Exchangis
+## 6. 如何登录使用 Exchangis
 
-Exchangis1.0更多使用说明，请参考用户使用手册[Exchangis1.0 用户手册](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0/docs/zh_CN/ch1/exchangis_user_manual_cn.md)
+Exchangis更多使用说明，请参考用户使用手册[Exchangis 用户手册](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.1.2/docs/zh_CN/ch1/exchangis_user_manual_cn.md)
```
Comment (wushengyeyouya) [inline docs/zh_CN/ch1/exchangis_deploy_cn.md:308]: please modify it,

File: docs/zh_CN/ch1/exchangis_sqoop_deploy_cn.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 ### 环境准备
 Sqoop引擎是执行Exchangis数据同步任务不可或缺的组件，只有安装部署完成Sqoop引擎才能够成功执行数据同步任务。同时，确保所部署的机器上有安装sqoop。
 
-您在安装部署Sqoop引擎之前，请按照[Exchangis1.0.0安装部署文档](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0/docs/zh_CN/ch1/exchangis_deploy_cn.md)安装完成Exchangis1.0.0及相关组件的安装，并确保工程基本功能可用。
+您在安装部署Sqoop引擎之前，请按照[Exchangis1.1.2安装部署文档](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.1.2/docs/zh_CN/ch1/exchangis_deploy_cn.md)安装完成Exchangis1.1.2及相关组件的安装，并确保工程基本功能可用。
```
Comment (wushengyeyouya) [inline docs/zh_CN/ch1/exchangis_sqoop_deploy_cn.md:5]: please modify it.

**Review by wushengyeyouya — CHANGES_REQUESTED** [review]

File: docs/en_US/ch1/exchangis_deploy_en.md [hunk 5]
```diff
@@ -191,7 +214,7 @@ As shown in the figure below:
 
 #### 2.7.1  Get the front-end installation package 
 
-Exchangis has provided compiled front-end installation package by default, which can be downloaded and used directly ：[Click to jump to the Release interface](https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/WeDatasphere/Exchangis/exchangis1.0.0/dist.zip)
+Exchangis has provided compiled front-end installation package by default, which can be downloaded and used directly ：[Click to jump to the Release interface](https://github.com/WeBankFinTech/Exchangis/releases/download/release-1.0.0-rc1/web-dist.zip)
```
Comment (wushengyeyouya) [inline docs/en_US/ch1/exchangis_deploy_en.md:217]: Must modify it.

File: docs/en_US/ch1/exchangis_deploy_en.md [hunk 6]
```diff
@@ -13,26 +13,26 @@ Exchangis installation is mainly divided into the following four steps ：
...
-| MySQL (5.5+)                                                                 | yes | [How to install mysql](https://www.runoob.com/mysql/mysql-install.html) |
-| JDK (1.8.0_141)                                                              | yes | [How to install JDK](https://www.runoob.com/java/java-environment-setup.html) |
-| Hadoop(2.7.2，Other versions of Hadoop need to compile Linkis by themselves.) | yes | [Hadoop stand-alone deployment](https://hadoop.apache.org/releases.html) ；[Hadoop distributed deployment](https://hadoop.apache.org/releases.html) |
-| Hive(2.3.3，Other versions of Hive need to compile Linkis by themselves.)     | yes | [Hive quick installation](https://hive.apache.org/downloads.html) |
+| JDK (1.8.0_141)                                                  | yes | [How to install JDK](https://www.oracle.com/java/technologies/downloads/) |
+| MySQL (5.5+)                                                  | yes | [How to install mysql](https://mysql.net.cn/) |
+| Hadoop(3.3.4，Other versions of Hadoop need to compile Linkis by themselves.) | yes | [Hadoop deployment](https://www.apache.org/dyn/closer.cgi/hadoop/common/hadoop-3.3.4/hadoop-3.3.4.tar.gz) |
```
Comment (wushengyeyouya) [inline docs/en_US/ch1/exchangis_deploy_en.md:18]: hadoop3.3.4 is sure?

File: docs/en_US/ch1/exchangis_deploy_en.md [hunk 7]
```diff
@@ -273,12 +296,12 @@ Please visit the Exchange front-end page at  http://${EXCHANGIS_INSTALL_IP}:8098
 
 ## 3. DSS ExchangisAppConn installation and deployment 
 
-If you want to use Exchangis1.0.0 front-end, you also need to install the DSS ExchangisAppConn plugin. Please refer to: [ExchangisAppConn installation documentation for plugins ](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0/docs/en_US/ch1/exchangis_appconn_deploy_en.md)
+If you want to use Exchangis1.0.0 front-end, you also need to install the DSS ExchangisAppConn plugin. Please refer to: [ExchangisAppConn installation documentation for plugins ](./docs/en_US/ch1/exchangis_appconn_deploy_en.md)
 
 ## 4. Linkis Sqoop engine installation and deployment 
 
-If you want to execute the Sqoop operation of Exchangis1.0.0 normally, you also need to install the Linkis Sqoop engine. Please refer to: : [Linkis Sqoop engine installation documentation ](https://linkis.apache.org/zh-CN/docs/latest/engine-usage/sqoop)
+If you want to execute the Sqoop operation of Exchangis1.0.0 normally, you also need to install the Linkis Sqoop engine. Please refer to: : [Linkis Sqoop engine installation documentation ](https://linkis.apache.org/zh-CN/docs/1.1.2/engine-usage/sqoop/)
```
Comment (wushengyeyouya) [inline docs/en_US/ch1/exchangis_deploy_en.md:303]: not the latest linkis version?

File: docs/zh_CN/ch1/component_upgrade_cn.md [hunk 8]
```diff
@@ -14,7 +14,7 @@ Exchangis 升级文档
 ```
 
 #### 2）下载二进制包
-我们提供ExchangisAppconn的升级物料包，您可直接下载使用。[点击跳转 Release 界面](https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/WeDatasphere/Exchangis/exchangis1.0.0-rc/exchangis-appconn.zip)
+我们提供ExchangisAppconn的升级物料包，您可直接下载使用。[点击跳转 Release 界面](https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/WeDatasphere/Exchangis/exchangis1.1.2/Exchangis1.1.2_install_package.zip)（TODO：待发版时更新）
```
Comment (wushengyeyouya) [inline docs/zh_CN/ch1/component_upgrade_cn.md:17]: please delete 'TODO'

File: docs/zh_CN/ch1/exchangis_deploy_cn.md [hunk 9]
```diff
@@ -6,33 +6,34 @@ Exchangis 的安装，主要分为以下四步：
...
 
 | 依赖的组件                                 | 是否必装 | 安装直通车 |
 |---------------------------------------| ------ | --------------- |
-| MySQL (5.5+)                          | 必装  | [如何安装mysql](https://www.runoob.com/mysql/mysql-install.html) |
-| JDK (1.8.0_141)                       | 必装 | [如何安装JDK](https://www.runoob.com/java/java-environment-setup.html) |
-| Hadoop(2.7.2，Hadoop 其他版本需自行编译 Linkis) | 必装 | [Hadoop单机部署](https://hadoop.apache.org/releases.html) ；[Hadoop分布式部署](https://hadoop.apache.org/releases.html) |
-| Hive(2.3.3，Hive 其他版本需自行编译 Linkis)     | 必装 | [Hive快速安装](https://hive.apache.org/downloads.html) |
+| JDK (1.8.0_141)           | 必装  | [如何安装JDK](https://www.oracle.com/java/technologies/downloads/) |
+| MySQL (5.5+)           | 必装 | [如何安装mysql](https://www.runoob.com/mysql/mysql-install.html) |
+| Hadoop(3.3.4，Hadoop 其他版本需自行编译 Linkis) | 必装 | [Hadoop部署](https://www.apache.org/dyn/closer.cgi/hadoop/common/hadoop-3.3.4/hadoop-3.3.4.tar.gz) |
```
Comment (wushengyeyouya) [inline docs/zh_CN/ch1/exchangis_deploy_cn.md:19]: hadoop3.3.4 is sure?

File: docs/zh_CN/ch1/exchangis_deploy_cn.md [hunk 10]
```diff
@@ -271,12 +293,16 @@ Exchangis 已默认提供了编译好的前端安装包，可直接下载使用
 
 ## 3. DSS ExchangisAppConn 安装部署
 
-如您想正常使用 Exchangis1.0.0 前端，还需安装 DSS ExchangisAppConn 插件，请参考: [ExchangisAppConn 插件安装文档](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.0.0/docs/zh_CN/ch1/exchangis_appconn_deploy_cn.md)
+如您想正常使用 Exchangis1.1.2 前端，还需安装 DSS ExchangisAppConn 插件，请参考: [ExchangisAppConn 插件安装文档](./docs/zh_CN/ch1/exchangis_appconn_deploy_cn.md)
 
 ## 4. Linkis Sqoop 引擎安装部署
 
-如您想正常执行 Exchangis1.0.0 的 Sqoop作业，还需安装 Linkis Sqoop 引擎，请参考: [Linkis Sqoop 引擎插件安装文档](https://linkis.apache.org/zh-CN/docs/latest/engine-usage/sqoop)
+如您想正常执行 Exchangis1.1.2 的 Sqoop作业，还需安装 Linkis Sqoop 引擎，请参考: [Linkis Sqoop 引擎插件安装文档](https://github.com/WeBankFinTech/Exchangis/blob/dev-1.1.2/docs/zh_CN/ch1/exchangis_sqoop_deploy_cn.md)
```
Comment (wushengyeyouya) [inline docs/zh_CN/ch1/exchangis_deploy_cn.md:300]: Please modify it.

**Review by wushengyeyouya — CHANGES_REQUESTED** [review]

File: README-ZH.md
[the remark this answers is not in the document]
Comment (jefftlin) [reply README-ZH.md:1]: The file has been updated to the latest version.

File: README.md
[the remark this answers is not in the document]
Comment (jefftlin) [reply README.md:1]: The file has been updated to the latest version.

File: docs/zh_CN/ch1/exchangis_deploy_cn.md (hunk 2 above)
Comment (jefftlin) [reply docs/zh_CN/ch1/exchangis_deploy_cn.md:304]: It has been changed to no version number.

File: docs/en_US/ch1/exchangis_appconn_deploy_en.md (hunk 1 above)
Comment (jefftlin) [reply docs/en_US/ch1/exchangis_appconn_deploy_en.md]: It has been changed to the relative path.

File: docs/zh_CN/ch1/exchangis_deploy_cn.md (hunk 3 above)
Comment (jefftlin) [reply docs/zh_CN/ch1/exchangis_deploy_cn.md]: It has been changed to the relative path.

File: docs/zh_CN/ch1/exchangis_sqoop_deploy_cn.md (hunk 4 above)
Comment (jefftlin) [reply docs/zh_CN/ch1/exchangis_sqoop_deploy_cn.md]: It has been changed to no version number.

File: docs/en_US/ch1/exchangis_deploy_en.md (hunk 5 above)
Comment (jefftlin) [reply docs/en_US/ch1/exchangis_deploy_en.md]: It has been updated with the latest link.

File: docs/en_US/ch1/exchangis_deploy_en.md (hunk 6 above)
Comment (jefftlin) [reply docs/en_US/ch1/exchangis_deploy_en.md:18]: Hadoop was upgraded to 3.3.4 as with linkis1.4.0.

File: docs/en_US/ch1/exchangis_deploy_en.md (hunk 7 above)
Comment (jefftlin) [reply docs/en_US/ch1/exchangis_deploy_en.md:303]: It has been updated with the latest link.

File: docs/zh_CN/ch1/component_upgrade_cn.md (hunk 8 above)
Comment (jefftlin) [reply docs/zh_CN/ch1/component_upgrade_cn.md:17]: It has been changed to the relative path.

File: docs/zh_CN/ch1/exchangis_deploy_cn.md (hunk 9 above)
Comment (jefftlin) [reply docs/zh_CN/ch1/exchangis_deploy_cn.md:19]: Hadoop was upgraded to 3.3.4 as with linkis1.4.0.

File: docs/zh_CN/ch1/exchangis_deploy_cn.md (hunk 10 above)
Comment (jefftlin) [reply docs/zh_CN/ch1/exchangis_deploy_cn.md:300]: It has been changed to the relative path.

**Review by wushengyeyouya — APPROVED** [review]
LGTM.

**Pull request merged**